src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -502,14 +502,22 @@
 
     <xsd:complexType name="transition">
         <xsd:sequence>
-            <xsd:element name="from" type="xsd:string" minOccurs="1" maxOccurs="unbounded" />
-            <xsd:element name="to" type="xsd:string" minOccurs="1" maxOccurs="unbounded" />
+            <xsd:element name="from" type="arc" minOccurs="1" maxOccurs="unbounded" />
+            <xsd:element name="to" type="arc" minOccurs="1" maxOccurs="unbounded" />
             <xsd:element name="metadata" type="metadata" minOccurs="0" maxOccurs="unbounded" />
             <xsd:element name="guard" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
         </xsd:sequence>
         <xsd:attribute name="name" type="xsd:string" use="required" />
     </xsd:complexType>
 
+    <xsd:complexType name="arc">
+        <xsd:simpleContent>
+            <xsd:extension base="xsd:string">
+                <xsd:attribute name="weight" type="xsd:integer" use="optional" />
```

Comment:
`use="optional"` is the default and we generally don't put it explicitly in our schemas